tests/unit_tests/test_aws.py

```diff
@@ -0,0 +1,23 @@
+from unittest.mock import patch
+
+import pytest
+
+from sky.clouds.aws import AWS
+
+
+def test_aws_label():
+    aws = AWS()
+    # Invalid - AWS prefix
+    assert aws.is_label_valid("aws:whatever", "value")[0] == False
```

Comment:
style nit: we don't compare boolean with `==`. 
Reference: https://google.github.io/styleguide/pyguide.html#2144-decision
```suggestion
    assert not aws.is_label_valid("aws:whatever", "value")[0]
```

Reply:
Done